Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/Logics/PinLogic.cs

```diff
@@ -242,10 +243,10 @@ protected override void OnUpdateType(Pin outerItem, Marker nativeItem)
 
         protected override void OnUpdateIcon(Pin outerItem, Marker nativeItem)
         {
-            if (outerItem.Icon != null && outerItem.Icon.Type == BitmapDescriptorType.View) 
+            if (outerItem.Icon != null && outerItem.Icon.Type == BitmapDescriptorType.View)
```

Comment:
Please don't contain "not the purpose" changes.
Code cleaning is "not purpose" for this PR.